web-ui/src/components/reviews/periods/ReviewPeriodStepper.jsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import Stepper from '@mui/material/Stepper';
+import Step from '@mui/material/Step';
+import StepLabel from '@mui/material/StepLabel';
+
+const steps = [
+  'Review Period Is Open',
+  'Review Period Is Closed'
+];
+
+export default function HorizontalLinearAlternativeLabelStepper({reviewPeriod}) {
+  return (
+    <Box sx={{ width: '100%' }}>
+      <Stepper activeStep={reviewPeriod?.open === true ? 0 : 1} alternativeLabel>
```

Comment:
Checking for whether the review period status 'open' is true or false. If true, show the activeStep as being option 0 in the steps array, otherwise show option 1 as active.